game/unicode.hh

```diff
@@ -16,8 +17,8 @@ struct UnicodeUtil {
 	static std::string getCharset(std::string const& str);
 	static void convertToUTF8 (std::stringstream &_stream, std::string _filename);
 	static std::string convertToUTF8 (std::string const& str);
-	static std::string toLower (std::string const& str, size_t length = 0);
-	static std::string toUpper (std::string const& str, size_t length = 0);
+	static std::string toLower (std::string const& str, int length = 0);
+	static std::string toUpper (std::string const& str, int length = 0);
```

Comment:
len has conceptualy no reason to be negative, thus this would deserve to remain unsigend (moreover, if signed, the function is now buggy because we don't check for negative values)

Reply:
I changed that because the underlying ICU function takes int.